Source/APIClient/Request.swift

```diff
@@ -34,15 +36,22 @@ public protocol Request {
 
 public extension Request {
 
+    var contentType: String {
+        switch self {
+        default:
+            return "application/json"
+        }
+    }
+
     var urlRequest: URLRequest {
         let baseURL = Foundation.URL(string: Self.baseURL)!
-        let url = Foundation.URL(string: path, relativeTo: baseURL)!
+        let url = Foundation.URL(string: path, relativeTo: baseURL) ?? baseURL
```

Comment:
Right, this was an accommodation for the flickr api since it didn't use paths appended to the base URL. But yeah if it doesn't make sense for most APIs we use, then I'll happily revert it.